feat(alerting): Add RESULT_CONDITIONS in custom alert to have more information

aalmenar opened the pull request:
## Summary
 This helps solving issue: #1085 

Since this adds a new placeholder shouldn't interfere with the current behaviour gatus has.


## Checklist
- [X] Tested to validate that the changes work as intended, if applicable.
- [X] Updated documentation in `README.md`, if applicable.

aalmenar commented:
Rebased patch after: fix(alerting): Escape custom result errors

soymgomez commented:
It would be nice to have this on Gatus! For us it is useful to integrate it with our alert system.

@TwiN  Could you take a look at it?

aalmenar commented:
> I'm working on a very large change that touches nearly half of the files, that's why I haven't looked at this yet FYI

Since 5.24.0 is released now, can we revisit this?

TwiN commented:
@aalmenar Thank you for the reminder!

TwiN commented:
Can you test your change? I see you checked that you tested the change, but I'm not seeing any tests 🧐

aalmenar commented:
Hi @TwiN, i have added a test for this feature, and squashed the commits into 1.

Hope its ok for finally merging it.

> Thank you! The main reason I want tests is in part to make sure it works, but also to make sure future changes don't break the new feature.
> 
> I'll have another look once you pushed the tests, thank you for your patience!
